data/human/variants.txt

```diff
@@ -640,8 +639,8 @@ ship "Container Transport" "Container Transport (Blaster)"
 
 ship "Container Transport" "Container Transport (Hai)"
 	outfits
-		"Sidewinder Missile Launcher" 2
```

Comment:
Like with the Star Queen; I'm quite dubious about most freighters and transports carrying launchers. Pairs of pods seem better suited to covering their flight to hyperspace than a single launcher.